mirrord/agent/src/steal/ip_tables.rs

```diff
@@ -117,6 +117,17 @@ pub struct IPTablesWrapper {
     tables: Arc<iptables::IPTables>,
 }
 
+/// wrapper around iptables::new that uses nft or legacy based on env
+pub fn new_iptables() -> Result<IPTables, Box<dyn Error>> {
```

Comment:
Do we have `Box<dyn Error>` anywhere else in the agent?

- If we do: then this gets a pass.
- If we don't: please wrap in an error.

Reply:
It's just a propagation of the underlying error, didn't want to change the whole code for that.